web/views/unit_view.ex

```diff
@@ -7,7 +7,7 @@ defmodule OrgtoolDb.UnitView do
   has_many :units,
     serializer: OrgtoolDb.UnitView,
     include: false,
-    identifiers: :when_included
+    identifiers: :always
```

Comment:
As mentioned above I'd rather keep them on 'when_included` and read them only when needed